Model/ZimFileService/ZimFileService.mm

```diff
@@ -115,6 +115,16 @@ + (ZimFileMetaData *)getMetaDataWithFileURL:(NSURL *)url {
     return metaData;
 }
 
+- (kiwix::Book) getBookBy:(nonnull NSUUID *) fileZimID {
+    kiwix::Book book = kiwix::Book();
+    try {
+        NSURL *fileURL = [ZimFileService.sharedInstance getFileURL: fileZimID];
+        book.update(zim::Archive([fileURL fileSystemRepresentation]));
+    } catch (std::exception e) { }
+    return book;
+}
```

Comment:
It is a lot of legwork at the moment:
- we need a `kiwix::Book` in order to be able create a `kiwix::Bookmark`
- we need a `kiwix::Archive` in order to be able to create `kiwix::Book`
Whereas all we want is to store:
- 2 strings (title, description) a thumbnail URL, keyed by the article URL, which is more or less this much code:
```
struct SwiftBookmark {
    let title: String
    let description: String
    let thumbnail: URL
}

final class SwiftBookmarkManager {
    private var dict: [URL: SwiftBookmark] = [:]
}
```